Source/Core/DolphinWX/ControllerConfigDiag.cpp

```diff
@@ -257,13 +257,13 @@ wxSizer* ControllerConfigDiag::CreateWiimoteConfigSizer()
 wxSizer* ControllerConfigDiag::CreatePassthroughBTConfigSizer()
 {
   m_passthrough_sync_text =
-      new wxStaticText(this, wxID_ANY, _("Sync real Wii Remotes and pair them"));
+      new wxStaticText(this, wxID_ANY, _("Sync Real Wii Remotes and Pair Them"));
   m_passthrough_sync_btn =
       new wxButton(this, wxID_ANY, _("Sync"), wxDefaultPosition, wxDLG_UNIT(this, wxSize(60, -1)));
   m_passthrough_sync_btn->Bind(wxEVT_BUTTON, &ControllerConfigDiag::OnPassthroughScanButton, this);
 
   m_passthrough_reset_text =
-      new wxStaticText(this, wxID_ANY, _("Reset all saved Wii Remote pairings"));
+      new wxStaticText(this, wxID_ANY, _("Reset All Saved Wii Remote Pairings"));
```

Comment:
I don't think there should be a colon, since it's very obviously a description for the button next to it, and the other labels in the Controller Configuration dialog don't have a colon either.